src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -33,7 +33,7 @@ public class LauncherSubsystem extends SubsystemBase {
 	private final SparkAbsoluteEncoder launcherAngleEncoder;
 	private final SparkPIDController launcherAnglePIDController;
 
-	private final GenericEntry launcherSpeedEntry =
+	private final GenericEntry SetlauncherSpeedEntry =
```

Comment:
Probably should be lowercase to start, to match the other class variables: `setlauncherSpeedEntry`